adam-cli/src/main/scala/org/bdgenomics/adam/cli/FlagStat.scala

```diff
@@ -41,6 +41,8 @@ class FlagStatArgs extends Args4jBase with ParquetArgs {
   val inputPath: String = null
   @Argument(required = false, metaVar = "OUTPUT", usage = "Optionally write the stats to this file.", index = 1)
   val outputPath: String = null
+  @Args4jOption(required = false, name = "-stringency", usage = "Set the parsing stringency: SILENT, LENIENT, STRICT.")
+  val stringency: String = "SILENT"
```

Comment:
what is the default if SAMHeaderReader.VALIDATION_STRINGENCY_PROPERTY is not set?